packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.tsx

```diff
@@ -10,9 +10,11 @@ import {
   getId,
   getNativeProps,
   inputProperties,
-  createRef
+  createRef,
+  hasOverflow
 } from '../../Utilities';
 import * as stylesImport from './ChoiceGroup.scss';
+import { TooltipHost, TooltipOverflowMode } from 'office-ui-fabric-react/lib/Tooltip';
```

Comment:
Bad import, you should import `../../ToolTip`. Fixing this should fix screener. I actually wrote a VSCode extension to help with this but I just need to get to cleaning the cobwebs from it.

Reply:
Ok, awesome, thank you!